boards/olimexino-stm32/doc.txt

```diff
@@ -63,7 +63,7 @@ make BOARD=olimexino-stm32 term
 USART1 is used for flashing the board whereas USART2 is the serial Output.
 
 ## Supported Toolchains
-For using the ST Nucleo-F103RB board we strongly recommend the usage of the
+For using the OLIMEXINO STM32 board we strongly recommend the usage of the
 [GNU Tools for ARM Embedded Processors](https://launchpad.net/gcc-arm-embedded)
 toolchain.
```

Comment:
tbh I doubt this note is still necessary.
The STM32F103RB isn't the most recent chip, any old `arm-none-eabi-gcc` should do.

Reply:
Yes you're right. I inspired myself with the doc of the Nucleo F103RB